safety.sh

```diff
@@ -0,0 +1,18 @@
+#/bin/bash
+currenttime=`date +"%d-%b-%Y %H:%M:%S"`
```

Comment:
I'm a fan of adding `set -e` to the top of these kinds of scripts so they bail on failures (usually atleast, there's lots of caveats with `set -e`) instead of stumble along and start causing splash damage